horace_core/sqw/@sqw/sqw.m

```diff
@@ -65,7 +65,7 @@
         % The class providing brief description of a whole sqw file.
         main_header_ = main_header_cl();
 
-        experiment_info_ = Experiment();
+        experiment_info_ = []; %Experiment();
```

Comment:
So the array has the right type.
```suggestion
        experiment_info_ = Experiment.empty();
```

Reply:
But I don't want it to have the right type. I want it to fail as obviously as possible if this variable has not been reinitialised correctly, probably in the constructor.